internal/compiler/emitter_statements.go

```diff
@@ -611,6 +612,23 @@ func (em *emitter) emitImport(node *ast.Import, isTemplate bool) []*runtime.Func
 	pkg := node.Tree.Nodes[0].(*ast.Package)
 	funcs, vars, inits := em.emitPackage(pkg, false, node.Tree.Path)
 
+	// If importing specific identifiers with "for" clause, filter to only those.
```

Comment:
What is the purpose of this change? It's not documented in the commit message, and the issue seems to be fixed even without it.